.codepipeline/local-dev/50_xdebug.ini

```diff
@@ -1,5 +0,0 @@
-; Uncomment and configure mode https://xdebug.org/docs/all_settings#xdebug.mode
```

Comment:
Not sure why you don't need this anymore, without apparent other changes to build strategy?

Reply:
OK - I think this is signed of by 'cmfive_dev_tools.sh'
Does that also mean "cmfive-boilerplate/.codepipeline/docker/start.sh" needs weeding out of its attempted XDebug settings?